frontend/src/components/common/AppLayout/AppLayout.tsx

```diff
@@ -35,17 +37,36 @@ const AppLayout: FC<IAppLayoutProps> = ({ ...props }) => {
         />
         <Content className="flex">
           <Switch>
-            {routes.map(r =>
-              r.content ? (
-                <Route exact key={r.route.path} path={r.route.path}>
-                  <Row className="h-full pt-5 xs:pt-10 pb-20 flex w-full px-4">
-                    <Col span={0} lg={1} xxl={2}></Col>
-                    {r.content}
-                    <Col span={0} lg={1} xxl={2}></Col>
-                  </Row>
-                </Route>
-              ) : null
+            {errorsQueue.length ? (
+              <div className="flex justify-center w-full items-center">
+                <Result
+                  status="500"
+                  title="Server error"
+                  subTitle={`Sorry, something went wrong. You have ${errorsQueue.length} errors in queue`}
+                />
+              </div>
+            ) : (
```

Comment:
Would add a `hasRenderingError` in the errorContext for this. Also I'd change the title with something like `Application error`, then as subTitle: `Sorry, something went wrong.` then it would be great if there's the possibility to add a button to refresh the page or go back to the home.

Reply:
It was just an experimental placeholder. I'll make these changes